VTEX - Checkout Configuration API.json

```diff
@@ -0,0 +1,350 @@
+                    {
+                        "$ref": "#/components/parameters/Content-Type"
+                    },
+                    {
+                        "$ref": "#/components/parameters/Accept"
+                    },
+                    {
+                        "name": "an",
+                        "in": "query",
+                        "description": "Checkout configuration ID.",
+                        "required": true,
+                        "schema": {
+                            "type": "string",
+                            "example": "12872182"
+                        }
+                    }
+                ],
+                "requestBody": {
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "$ref": "#/components/schemas/Settings"
+                            }
+                        }
+                    }
+                },
+                "responses": {
+                    "200": {
+                        "description": "Successful operation",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/Settings"
+                                }
+                            }
+                        }
+                    },
+                    "404": {
+                        "description": "Configuration not found"
+                    }
+                }
+            },
+            "delete": {
+                "tags": [
+                    "Configuration"
+                ],
+                "summary": "Delete checkout configuration",
+                "description": "Deletes an existing checkout configuration for the store. This endpoint allows clients to remove a checkout configuration that is no longer needed or has become obsolete.",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/Content-Type"
+                    },
+                    {
+                        "$ref": "#/components/parameters/Accept"
+                    },
+                    {
+                        "name": "an",
+                        "in": "query",
+                        "description": "Checkout configuration ID.",
+                        "required": true,
+                        "schema": {
+                            "type": "string",
+                            "example": "12872182"
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Successful operation"
+                    },
+                    "404": {
+                        "description": "Configuration not found"
+                    }
+                }
+            }
+        }
+    },
```

Comment:
Please add `security` here, otherwise the declared securitySchemes will not apply to any endpoint

```json
    "security": [
        {
            "appKey": [],
            "appToken": []
        },
        {
            "VtexIdclientAutCookie": []
        }
    ],
```